TYPO3.Neos/Configuration/NodeTypes.yaml

```diff
@@ -1,6 +1,6 @@
 # Base node, which just configures the "removed" property of the node.
 'TYPO3.Neos:Node':
-  label: "${String.cropAtWord(String.trim(String.stripTags(q(node).property('title') || q(node).property('text') || ((I18n.translate(node.nodeType.label) || node.nodeType.name) + (node.autoCreated ? ' (' + node.name + ')' : '')))), 100, '...')}"
+  label: "${String.cropAtWord(String.trim(String.stripTags(String.pregReplace(q(node).property('title') || q(node).property('text') || ((I18n.translate(node.nodeType.label) || node.nodeType.name) + (node.autoCreated ? ' (' + node.name + ')' : '')), '\/<br\\W*?\\/?>|\\x{00a0}|[[^:print:]]|\\s+\/u', ' '))), 100, '...')}"
```

Comment:
replaces `<br />` tags with spaces, removes non-breaking spaces, non-printable characters and removes double whitespace